client/app.html

```diff
@@ -43,7 +44,7 @@
     <script src="services/code_preprocessors/CodePreprocessorDispatcherService.js"></script>
     <script src="services/code_preprocessors/PythonCodePreprocessorService.js"></script>
   </head>
-  <body>
+  <body class="{{bodyClass}}">
```

Comment:
Yeah, it required a wrapper around the top level html object in the learner-view directive (also is why I had to add the height attribute a couple of places), but no reference to rootScope anymore!